File: Modul 3/C/PRAK301-2310817220025-FirdaKhoirunisa.c
#include <stdio.h>

int main(){
    int satu, dua;
    printf("Input\n");
    scanf("%d %d", &satu, &dua);

    int rendah, tinggi;
    if(satu <= dua){
        rendah = satu;
        tinggi = dua;
    }else{
        rendah = dua;
        tinggi = satu;
    }
    printf("\nOutput\n");
    printf("%d %d", rendah, tinggi);

    return 0;
}

File: Modul 3/Python/PRAK301-2310817220025-FirdaKhoirunisa.py
print("Input")
satu,dua = map(int, input().split())

if (satu <= dua):
    rendah = satu
    tinggi = dua
elif (dua <= satu):
    rendah = dua
    tinggi = satu

print("\nOutput")
print(rendah, tinggi)    